src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -221,7 +237,8 @@ def init_sync_d_interface_tables(db_conn):
     if_name_map = {if_name: sai_id for if_name, sai_id in if_name_map.items() if \
                    (re.match(port_util.SONIC_ETHERNET_RE_PATTERN, if_name.decode()) or \
                     re.match(port_util.SONIC_ETHERNET_BP_RE_PATTERN, if_name.decode()))}
-    if_id_map = {sai_id: if_name for sai_id, if_name in if_id_map.items() if \
+    # sai_id_key = namespace : sai_id
```

Comment:
>: [](start = 29, length = 1)

This is a new assumption that ':' will not appear in name space or sai_id. Please add code comment.

Reply:
How about using '|' which is widely used in Redis keys

---
In reply to: [453139692](https://github.com/Azure/sonic-snmpagent/pull/141#discussion_r453139692) [](ancestors = 453139692)

Reply:
Added comment.